test/core/security/jwt_verifier_test.cc

```diff
@@ -369,28 +363,27 @@ static void test_jwt_verifier_google_email_issuer_success(void) {
                                  nullptr);
   grpc_auth_json_key_destruct(&key);
   GPR_ASSERT(jwt != nullptr);
-  grpc_jwt_verifier_verify(&exec_ctx, verifier, nullptr, jwt, expected_audience,
+  grpc_jwt_verifier_verify(verifier, nullptr, jwt, expected_audience,
                            on_verification_success, (void*)expected_user_data);
-  grpc_jwt_verifier_destroy(&exec_ctx, verifier);
-  grpc_exec_ctx_finish(&exec_ctx);
+  grpc_jwt_verifier_destroy(verifier);
+  grpc_core::ExecCtx::Get()->Flush();
```

Comment:
Some grpc_exec_ctx_finish calls cannot just be eliminated. They need to be replaced with Flush. Google3 might have such instances too